src/gui/FxMixerView.cpp

```diff
@@ -395,6 +395,40 @@ void FxMixerView::deleteChannel(int index)
 
 
 
+void FxMixerView::deleteUnusedChannels()
+{
+	TrackContainer::TrackList tracks;
+	tracks += Engine::getSong()->tracks();
+	tracks += Engine::getBBTrackContainer()->tracks();
+
+	// go through all FX Channels
+	for(int i = m_fxChannelViews.size()-1; i>0; --i)
```

Comment:
coding style: always put spaces between operators and values
i > 0 // right
i>0 // wrong